src/BloomExe/NavigationIsolator.cs

```diff
@@ -148,11 +148,21 @@ private void BrowserOnDocumentCompleted(object sender, EventArgs eventArgs)
 		private void ForceDocumentCompleted()
 		{
 			Cleanup();
-			if (_pending.Count > 0)
+			while (_pending.Count > 0)
 			{
 				var task = _pending[0];
 				_pending.RemoveAt(0);
+				// Somehow raw files in /tmp can disappear after being sent to NavigationIsolator.Navigate() and before
+				// being displayed via the pending mechanism.  If the file doesn't exist, calling _current.Browser.Navigate()
+				// results in the program silently stopping on Linux.  (Going "poof" is how I would describe it.)
+				// See https://jira.sil.org/browse/BL-863 for a bug report.
+				if (task.Url.StartsWith("/tmp/") && !System.IO.File.Exists(task.Url))
+				{
+					Debug.WriteLine(String.Format("DEBUG: NavigationIsolator.ForceDocumentCompleted(): new file to display (\"{0}\") does not exist!??", task.Url));
```

Comment:
I wonder if we want a Debug.Assert here instead? At least on windows, we'll never notice this again, even if it is key to some other bug.